### PR TITLE
[IA-2295] Bump galaxy chart version

### PR DESCRIPTION
**Pull request by rtitle**
Includes Galaxy performance fixes plus a chart fix to run Galaxy jobs on the correct nodepool.

Will test in a fiab.

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala [hunk 1]
```diff
@@ -820,6 +820,7 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       // Node selector config: this ensures the app is run on the user's nodepool
       raw"""galaxy.nodeSelector.cloud\.google\.com/gke-nodepool=${nodepoolName.value}""",
       raw"""nfs.nodeSelector.cloud\.google\.com/gke-nodepool=${nodepoolName.value}""",
+      raw"""galaxy.configs.job_conf\.yml.runners.k8s.k8s_node_selector=cloud.google.com/gke-nodepool: ${nodepoolName.value}""",
```
Comment (rtitle) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala:823]: Corresponds to https://github.com/galaxyproject/galaxykubeman-helm/blob/master/galaxykubeman/values.yaml#L139

It's a little weird in that the property value is a string with a colon. I'm going to manually check that it is set correctly in k8s.

Comment (kyuksel) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala:823]: Is that `space` after `gke-nodepool:` supposed to be there?

**Review by kyuksel — APPROVED** [review]
LGTM if the one thing I mentioned inline is not an issue and the FIAB testing works

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala (hunk 1 above)
Comment (rtitle) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala:823]: I think it is, not sure if it should be quoted here though -- will test

Comment (rtitle) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala:823]: I think this is actually correct, Galaxy starts up in my fiab and the conf looks correct to me:
```
    runners:
      k8s:
        k8s_cleanup_job: onsuccess
        k8s_fs_group_id: "101"
        k8s_galaxy_instance_id: 't08w2e-gxy-rls'
        k8s_job_api_version: batch/v1
        k8s_namespace: 't08w2e-gxy-ns'
        k8s_node_selector: 'cloud.google.com/gke-nodepool: k4173ee2-16cd-42e1-bd85-7f30913f88a7'
        k8s_persistent_volume_claims: |-
          t08w2e-gxy-rls-galaxy-pvc:/galaxy/server/database,t08w2e-gxy-rls-galaxy-cvmfs-gxy-data-pvc:/cvmfs/data.galaxyproject.org,t08w2e-gxy-rls-galaxy-cvmfs-gxy-main-pvc:/cvmfs/main.galaxyproject.org
        k8s_pod_priority_class: 't08w2e-gxy-rls-galaxy-job-priority'
        k8s_pod_retries: "4"
        k8s_pull_policy: IfNotPresent
        k8s_run_as_group_id: "101"
        k8s_run_as_user_id: "101"
        k8s_supplemental_group_id: "101"
        k8s_use_service_account: true
        k8s_walltime_limit: "604800"
        load: galaxy.jobs.runners.kubernetes:KubernetesJobRunner
      local:
        load: galaxy.jobs.runners.local:LocalJobRunner
        workers: 4
```

**Comment by rtitle** [comment]
jenkins retest

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala (hunk 1 above)
Comment (kyuksel) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala:823]: yay 👍

Comment (rtitle) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala:823]: However it looks like deletion does not work in this version, the `pre-delete-job` is failing. I pinged Alex and Dannon.

**Pull request merged**